src/Symfony/Component/Security/Core/Tests/SecurityTest.php

```diff
@@ -33,7 +41,7 @@ public function testGetToken()
 
         $container = $this->createContainer('security.token_storage', $tokenStorage);
 
-        $security = new Security($container);
+        $security = new Security([], $container);
```

Comment:
This should be undone, illustrating backwards compatibility. Similarly below.

Reply:
Thanks ! Changes done.